### PR TITLE
Add sideeffect

### PR DESCRIPTION
**Pull request by Justus-Saringer**

### CODE REVIEW

File: code/sideeffects/SideEffect.kt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            .size(100.dp)
+            .padding(20.dp)
+            .background(Green200),
+        contentAlignment = Alignment.Center
+    ) {
+        Text(
+            text = "$timer",
+        )
+    }
+
```
Comment (Darius3001) [inline code/sideeffects/SideEffect.kt:23]: could you maybe add a comment what problem will occur

File: code/sideeffects/SideEffect.kt [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            .size(100.dp)
+            .padding(20.dp)
+            .background(Green200),
+    ) {
+        Text(
+            text = "$timer",
+        )
+    }
+
+    SideEffect {
```
Comment (Darius3001) [inline code/sideeffects/SideEffect.kt:46]: A lil comment on what it does. mustnt be to long

File: demo/src/main/java/de/check24/compose/demo/features/sideeffects/ComposableSideEffectActivity.kt [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+            .padding(20.dp)
+            .background(Green200),
+        contentAlignment = Alignment.Center
+    ) {
+        Text(
+            text = "$timer",
+        )
+    }
+
+    Thread.sleep(1000)
```
Comment (Darius3001) [inline demo/src/main/java/de/check24/compose/demo/features/sideeffects/ComposableSideEffectActivity.kt:87]: you can also apply the comments in the demo

**Review by Darius3001 — CHANGES_REQUESTED** [review]

File: demo/src/main/java/de/check24/compose/demo/features/sideeffects/ComposableSideEffectActivity.kt (hunk 3 above)
Comment (Justus-Saringer) [reply demo/src/main/java/de/check24/compose/demo/features/sideeffects/ComposableSideEffectActivity.kt:91]: I added comments to both parts

**Review by Darius3001 — APPROVED** [review]

**Pull request merged**